test/parser.js

```diff
@@ -26,6 +26,14 @@ describe('<message>', () => {
       parsed = parser('feat(http parser)!: add support for scopes')
       parsed.should.matchSnapshot()
     })
+    it('parses summary with multiple spaces after separator', () => {
```

Comment:
I think this change to the parser is great.

Mind updating the grammar in the README too, adding the permissive `<whitespace>*` marker wherever it's now supported?

Reply:
It was already in the grammar actually @bcoe 

- `<summary>` may include [`":", <whitespace>*`](https://github.com/conventional-commits/parser/blame/main/README.md#L66-L67)
- `<footer>` may include [`<separator>, <whitespace>*`](https://github.com/conventional-commits/parser/blame/main/README.md#L80)

The only change I made was switching [`<continuation>` from `<whitespace>`](https://github.com/conventional-commits/parser/blame/main/README.md#L87) to `<whitespace>+`